pkg/plugin/trivy/plugin.go

```diff
@@ -254,20 +282,31 @@ func NewPlugin(clock ext.Clock, idGenerator ext.IDGenerator, client client.Clien
 	}
 }
 
+// NewTrivyConfigAuditPlugin constructs a new configAudit.Plugin, which is using an
+// upstream Trivy config audit scanner lib.
+func NewTrivyConfigAuditPlugin(clock ext.Clock, idGenerator ext.IDGenerator, client client.Client) configauditreport.PluginInMemory {
```

Comment:
Shouldn't it be only one plugin? `trivy` and it supports different types of scans?

Reply:
eventually I agree with you however in order to make the change gradually , need to go step by step.
I opened a task to clean up prev. logic for configaudit plugin  #192